ui/components/app/transaction-detail-item/index.scss

```diff
@@ -22,9 +29,10 @@
     }
   }
 
-  &__total {
+  & h6.transaction-detail-item__total {
     font-weight: bold;
     color: $ui-black;
+    margin-left: 4px;
   }
```

Comment:
Change above is to make this rule more specific.

Reply:
Hmm. We should be able to write CSS in a way that doesn't require these kinds of specificity overrides. What were you needing to override?

Reply:
Yep I agree this should ideally not be needed, this classes are styled in couple of components - that makes it hard to fix. Let me spend some more time on it.

Reply:
It is fixed now and code looks cleaner :)